docs/dev/table/sql.md

```diff
@@ -1510,6 +1510,17 @@ ATAN(numeric)
       </td>
     </tr>
 
+    <tr>
+      <td>
+        {% highlight text %}
+ATAN2(numeric, numeric)
+{% endhighlight %}
+      </td>
+      <td>
+        <p>Calculates the arc tangent of a given coordinate.</p>
```

Comment:
I checked the [wiki](https://en.wikipedia.org/wiki/Atan2) about atan2, it said: The atan2 function calculates one unique arc tangent value from two variables y and x. So, would it be better `of two variables`?  atan2(y,x) can be the arc tangent of (x,y) or (nx, ny).

Reply:
Hello @hequn8128! The idea about coordinates comes from Math.atan2 + Calcite's definition here [1] ATAN2(numeric, numeric) | Returns the arc tangent of the numeric coordinates.
Does it make sense to take the same wording as Calcite has?
[1] https://calcite.apache.org/docs/reference.html